Style-Guide/Code-Layout-and-Formatting.md

```diff
@@ -82,12 +82,21 @@ end{}
 You can always delete or ignore one of the blocks (or add the `begin` block), add parameters and so on, but you should avoid writing scripts or functions without CmdletBinding, and you should always at least _consider_ making it take pipeline input.
 
 #### Open braces on the same line
```

Comment:
Should be: 
#### Follow the one-true-brace style.
##### Open braces always go on the same line
...
##### Close braces start a new line ... but follow-on statements like `else` stay on that line.